src/v2/Apps/PriceDatabase/Components/PriceDatabaseBenefits.tsx

```diff
@@ -0,0 +1,84 @@
+import {
+  Column,
+  GridColumns,
+  Text,
+  Image,
+  Separator,
+  Flex,
+  themeProps,
+} from "@artsy/palette"
+import React from "react"
+import { useMatchMedia } from "v2/Utils/Hooks/useMatchMedia"
+
+export const PriceDatabaseBenefits: React.FC = () => {
+  const isMobile = useMatchMedia(themeProps.mediaQueries.xs)
+
+  return (
+    <Flex py={[1, 4]} flexDirection="column">
+      <GridColumns my={4} gridRowGap={[2, 0]}>
+        <Column span={12}>
+          <Text as="h1" variant="xl">
+            The Benefits of the
+            <br />
+            Artsy Price Database
+          </Text>
+        </Column>
+      </GridColumns>
+      <GridColumns mb={4} gridRowGap={[2, 0]}>
+        <Column span={6}>
+          <Image style={{ maxWidth: "100%" }} src={"images/browse.jpg"} />
```

Comment:
Where is this image coming from? For all images we should always have width, height, alt, srcSet.

Reply:
I'm still waiting for the final content of the page. This is just an example image and will be replaced later on. 

I wrapped the `Image` component in a `ResponsiveBox` to give it width, height, and so on.